src/libs/blackboard/qa/Makefile

```diff
@@ -50,25 +50,27 @@ LIBS_qa_bb_objpos = fawkescore fawkesblackboard fawkesinterface ObjectPositionIn
 			fawkesutils fawkesnetcomm fawkeslogging
 OBJS_qa_bb_objpos = qa_bb_objpos.o
 
-OBJS_all =	$(OBJS_qa_bb_memmgr) 		\
-		$(OBJS_qa_bb_interface)		\
-		$(OBJS_qa_bb_buffers)		\
-		$(OBJS_qa_bb_messaging)		\
-		$(OBJS_qa_bb_openall)		\
-		$(OBJS_qa_bb_notify)		\
-		$(OBJS_qa_bb_listall)		\
-		$(OBJS_qa_bb_remote)		\
-		$(OBJS_qa_bb_objpos)
-
-BINS_all =	$(BINDIR)/qa_bb_memmgr		\
-		$(BINDIR)/qa_bb_interface	\
-		$(BINDIR)/qa_bb_buffers		\
-		$(BINDIR)/qa_bb_messaging	\
-		$(BINDIR)/qa_bb_notify		\
-		$(BINDIR)/qa_bb_openall		\
-		$(BINDIR)/qa_bb_listall		\
-		$(BINDIR)/qa_bb_remote		\
-		$(BINDIR)/qa_bb_objpos
+OBJS_all =	$(OBJS_qa_bb_memmgr)       \
```

Comment:
What's the change here? Is it just indentation? It also looks like the subsequent items are not indented correctly.

Reply:
Yes, it's replacing tabs with spaces. Indentation looks as intended (this qa Makefile change was just folded into the blackboard update).

Reply:
But it looks like it's missing two spaces and some of the `\` are misaligned (you probably need to click on the link as the preview here is not monospace):

https://github.com/fawkesrobotics/fawkes/blob/9e9da11bba30ed5b31bc5f7e6a295af45b88cb20/src/libs/blackboard/qa/Makefile#L53-L71

Reply:
Indeed, some stray tabs (but still aligning properly here...). Just pushed a fix.